### PR TITLE
fix: uniform tile icons design update

### LINKED ISSUE #998: Improvements on Power Source Tile
**Actual Behaviour**

Tile image for power source looks big and mis-aligned

**Expected Behaviour**

Tile image needs to be aligned properly with correct margins around them, even in portrait mode.

**Steps to reproduce it**

1. Open app
2. Scroll down instruments list for `Power Source`

**LogCat for the issue**

> Not available

**Screenshots of the issue**

screenshot_2018-06-08-18-41-06

**Would you like to work on the issue?**

Yes

### LINKED ISSUE #1024: Improvements on Oscilloscope Tile
**Actual Behaviour**

Tile uses a background image which doesn't stretch out properly in different screens

**Expected Behaviour**

Tile layout needs to be designed with proper layouts like linear with weights to accommodate content in a proper order with indents and scaling features. This includes modifying the tile images to have a similar theme. (All having similar shape and same line widths)

**Steps to reproduce it**

Observe the `Instrument` list

**LogCat for the issue**

> Not available

**Screenshots of the issue**

> Not available

**Would you like to work on the issue?**

Yes

### LINKED ISSUE #1025: Improvements on Multimeter Tile
**Actual Behaviour**

Tile uses a background image which doesn't stretch out properly in different screens

**Expected Behaviour**

Tile layout needs to be designed with proper layouts like linear with weights to accommodate content in a proper order with indents and scaling features. This includes modifying the tile images to have a similar theme. (All having similar shape and same line widths)

**Steps to reproduce it**

Observe the `Instrument` list

**LogCat for the issue**

> Not available

**Screenshots of the issue**

> Not available

**Would you like to work on the issue?**

Yes

### LINKED ISSUE #1026: Improvements on Logic Analyzer Tile
**Actual Behaviour**

Tile uses a background image which doesn't stretch out properly in different screens

**Expected Behaviour**

Tile layout needs to be designed with proper layouts like linear with weights to accommodate content in a proper order with indents and scaling features. This includes modifying the tile images to have a similar theme. (All having similar shape and same line widths)

**Steps to reproduce it**

Observe the `Instrument` list

**LogCat for the issue**

> Not available

**Screenshots of the issue**

> Not available

**Would you like to work on the issue?**

Yes

### LINKED ISSUE #1027: Improvements on Sensors Tile
**Actual Behaviour**

Tile uses a background image which doesn't stretch out properly in different screens

**Expected Behaviour**

Tile layout needs to be designed with proper layouts like linear with weights to accommodate content in a proper order with indents and scaling features. This includes modifying the tile images to have a similar theme. (All having similar shape and same line widths)

**Steps to reproduce it**

Observe the `Instrument` list

**LogCat for the issue**

> Not available

**Screenshots of the issue**

> Not available

**Would you like to work on the issue?**

Yes

### LINKED ISSUE #1028: Improvements on Wave Generator Tile
**Actual Behaviour**

Tile uses a background image which doesn't stretch out properly in different screens

**Expected Behaviour**

Tile layout needs to be designed with proper layouts like linear with weights to accommodate content in a proper order with indents and scaling features. This includes modifying the tile images to have a similar theme. (All having similar shape and same line widths)

**Steps to reproduce it**

Observe the `Instrument` list

**LogCat for the issue**

> Not available

**Screenshots of the issue**

> Not available

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by CloudyPadmal**
Fixes #1024 
Fixes #1025 
Fixes #1026  
Fixes #1027 
Fixes #1028  
Fixes #998  

**Changes**: 
Designed tile icons to have a common dimensions and theme as requested in the weekly meeting.

**Screenshot/s for the changes**: 

|  |  |  |
| ----- | ----- | ----- | 
|  |  |  |

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [X] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [X] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [X] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [X] My code does not contain any extra lines or extra spaces
- [X] I have requested reviews from other members

**APK for testing**: 
[Instrument_Tiles.zip](https://github.com/fossasia/pslab-android/files/2104774/Instrument_Tiles.zip)

### CODE REVIEW

**Comment by CloudyPadmal** [comment]
@Avjeet Vikum has opened #1046 to research on a tile icon and adding it.

File: app/src/main/res/layout/application_list_item.xml [hunk 1]
```diff
@@ -3,50 +3,90 @@
...
             <ImageView
                 android:id="@+id/application_icon"
-                android:layout_width="match_parent"
-                android:layout_height="@dimen/image_height"
+                android:layout_width="@dimen/instrument_tile_icon_size"
+                android:layout_height="@dimen/instrument_tile_icon_size"
+                android:layout_alignParentBottom="true"
+                android:layout_alignParentEnd="true"
+                android:layout_alignParentRight="true"
+                android:layout_marginBottom="10dp"
```
Comment (Avjeet) [inline app/src/main/res/layout/application_list_item.xml]: You have left this one

Comment (CloudyPadmal) [reply app/src/main/res/layout/application_list_item.xml]: Ah yes! Missed that one!

Comment (CloudyPadmal) [reply app/src/main/res/layout/application_list_item.xml]: @Avjeet done! :+1:

**Pull request merged**